src/components/Header/HeaderControls.tsx

```diff
@@ -110,7 +110,12 @@ export function HeaderControls({ setIsCodeFlyoutVisible }: IHeaderControls) {
           </ControlButton>
         </EuiFlexItem>
       )}
-      <EuiFlexItem grow={recordingStatus === RecordingStatus.Recording}>
+      <EuiFlexItem
+        grow={
+          recordingStatus === RecordingStatus.Recording ||
+          recordingStatus === RecordingStatus.Paused
+        }
+      >
```

Comment:
Would this work? (I've tested locally but not sure if I missed some reasons behind it)
```suggestion
      <EuiFlexItem grow>
```

Reply:
No this won't work because we want it to `grow` only when status is `Recording` or `Paused`. When we are `NotRecording`, we want the URL input to expand to fill the rest of the space in this row. If we `grow` always, the input will not be able to fill the entire space:

<img width="1364" alt="image" src="https://user-images.githubusercontent.com/18429259/164074607-32754a83-5468-4f9d-980c-ee3063f5832b.png">

vs. if we use the code as written:

<img width="1358" alt="image" src="https://user-images.githubusercontent.com/18429259/164074668-f891b3df-1875-435f-90b1-d3e7e7610127.png">

Reply:
Sometimes they're a necessary evil 😆